snuba/admin/static/capacity_management/row_data.tsx

```diff
@@ -6,30 +6,46 @@ function getReadonlyRow(
   config: AllocationPolicyConfig,
   edit: () => void
 ): RowData {
-  return [
-    <code style={{ wordBreak: "break-all", color: "black" }}>
-      {config.name}
-    </code>,
-    <code style={{ wordBreak: "break-all", color: "black" }}>
-      {Object.keys(config.params).length
-        ? JSON.stringify(config.params)
-        : "N/A"}
-    </code>,
-    <code style={{ wordBreak: "break-all", color: "black" }}>
-      {config.value}
-    </code>,
-    <code style={{ wordBreak: "normal", overflowWrap: "anywhere", color: "black" }}>
-      {config.description}
-    </code>,
-    config.type,
-    <Button
-      variant="outline-secondary"
-      onClick={() => edit()}
-      data-testid={config.name + "_edit"}
-    >
-      edit
-    </Button>,
-  ];
+  return {
```

Comment:
is this just prettier?

Reply:
I think this makes it more readable, because this was a list before, so when feeding the result of this function into the <Table> element we need to remove an element by index. Making this an object shows explicitly what we information we are omitting.

Reply:
ah I see